### DISCUSSION TITLE
Is Grallvm latest version 22.2 supported?

### DISCUSSION THREAD

**Discussion by TheBlindM** [quarkusio/quarkus#27150] [Community]
Is Grallvm latest version 22.2 supported?

**Comment by gastaldi** [comment]
@TheBlindM it was merged in https://github.com/quarkusio/quarkus/pull/26943 and should land in 2.12.0.Final (due end of August - see https://github.com/quarkusio/quarkus/wiki/Release-Planning)

**Reply by gsmet** [reply]
Yeah, and it should already work with 2.11. I built a few Quarkus 2.11 apps with 22.2 already.
